### ISSUE TITLE
我启动vins后内存一直在增加直到占满，是不是哪里没写好内存溢出了啊？

### ISSUE THREAD

**Issue by yst-one** [HKUST-Aerial-Robotics/VINS-Fusion#212]

**Comment by ZW-PRO** [comment]
Have you solved this problem?

**Comment by ZW-PRO** [comment]
Have you solved this problem?

**Comment by JanekDev** [comment]
@ZW-PRO any updates on the problem? I have some minor observations, so we could possibly exchange info and work something out?

**Comment by ZW-PRO** [comment]
> @ZW-PRO any updates on the problem? I have some minor observations, so we could possibly exchange info and work something out?

Yes. I'm running VINS in ros2 and I've noticed that when I limit the size of the queue in the img0_callback() function, the memory doesn't increase anymore, but I guess this can cause issues like image loss. The probable reason is that the image processing speed in the queue cannot keep up with the image speed of the incoming queue.

**Comment by ZW-PRO** [comment]
> Referenc

Have you solved this problem?

**Comment by ZW-PRO** [comment]
@JanekDev Have you solved this problem?
